Documentation/install/rhel/installing-workers.md

```diff
@@ -123,14 +123,25 @@ Once this value has been retrieved it will be placed in the user managed file `/
 The default CNI installation for Tectonic uses VXLAN for its communications with [flannel][flannel-repo], which requires communications between hosts on UDP port 4789. The Kubernetes API also communicates with hosts on TCP port 10250. To simplify the configuration of these options, either allow all communications between cluster members, place the relevant ethernet interfaces into the "trusted" zone using FirewallD, or at a minimum allow `4789/udp` and `10250/tcp`. These last steps can be completed with the commands:
 
 ```
-$ firewall-cmd --add-port 10250/tcp
-$ firewall-cmd --add-port 10250/tcp --permanent
-$ firewall-cmd --add-port 4789/udp
-$ firewall-cmd --add-port 4789/udp --permanent
+$ sudo firewall-cmd --add-port 10250/tcp
+$ sudo firewall-cmd --add-port 10250/tcp --permanent
+$ sudo firewall-cmd --add-port 4789/udp
+$ sudo firewall-cmd --add-port 4789/udp --permanent
 ```
 
 Note: These settings may not be all inclusive and will not represent relative node ports or other communications which may need to be performed. For more information consult the [Kubernetes Networking][k8s-networking] documentation.
 
+Or alternatively you can allow all communication between cluster members, using "trusted" zones with relevant ethernet interfaces with the commands:
```

Comment:
Swap the order of these two sets of commands, and list this one first. paragraph on line 123 becomes 

... To simplify the configuration of these options, either allow all communications between cluster members, or allow `4789/udp` and `10250/tcp`.

Use firewalld to allow all communication between cluster members, using "trusted" zones with relevant ethernet interfaces:

``` example ```

then:

Or, allow `4789/udp` and `10250/tcp`.

``` example ```